pandas/io/html.py

```diff
@@ -73,7 +73,7 @@ def _importers():
 _RE_WHITESPACE = re.compile(r"[\r\n]+|\s{2,}")
 
 
-def _remove_whitespace(s: str, regex=_RE_WHITESPACE) -> str:
+def _remove_whitespace(s: str, regex=_RE_WHITESPACE, remove_whitespace=True) -> str:
```

Comment:
seems a bit odd to define a boolean parameter input to a function named `remove_whitespace` if it really wants to `remove_whitespace` or not. 
Better to do a conditional check before the function is called, and either call it or don't.

Reply:
excellent point. updated the pull to reflect this suggestion